optimistix/_solver/nelder_mead.py

```diff
@@ -171,6 +177,24 @@ def init(
                     + self.adelta
                     + self.rdelta * leaf[relative_indices]
                 )
+                if lower is not None:
+                    broadcast_leaves = jtu.tree_map(
+                        lambda a, b: jnp.clip(a, a_min=b),
+                        jnp.array(broadcast_leaves),
+                        jnp.broadcast_to(
+                            jnp.array(lower_bounds[index]),
+                            shape=jnp.array(broadcast_leaves).shape,
```

Comment:
I think the `jnp.array(broadcast_leaves)` should be unnecessary, from the code above it already looks to be an array?